content/3_Silver/Flood_Fill.mdx

```diff
@@ -739,3 +739,23 @@ print(room_num)
 ## Problems
 
 <Problems problems="general" hideSuggestProblemButton />
+
+## Quiz
+
+<Quiz>
+	<Quiz.Question>
+		What would be a valid function header for DFS on an undirected graph to work *without* any other variables?
```

Comment:
none of these? don't you need to keep track of what's visited and what's not?

Reply:
meant to say tree